modules/aggregation-layers/src/utils/bin-sorter.js

```diff
@@ -56,6 +57,19 @@ export default class BinSorter {
       .sort((a, b) => a.value - b.value);
   }
 
+  percentileToIndex([lowerPercentile, upperPercentile]) {
```

Comment:
Nit: individual array elements are not being used, so keep the array as is, `percentileToIndex(percentileRange)`